src/components/v5/common/CompletedAction/partials/WidgetContentLoaderSkeleton/WidgetContentLoaderSkeleton.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/LoadingSkeleton.tsx';
+import MenuWithStatusText from '~v5/shared/MenuWithStatusText/MenuWithStatusText.tsx';
+import { StatusTypes } from '~v5/shared/StatusText/consts.ts';
+
+export const WidgetContentLoaderSkeleton = () => (
+  <MenuWithStatusText
+    isLoading
+    statusTextSectionProps={{ status: StatusTypes.Info }}
+    sections={[
+      {
+        key: '1',
+        content: (
+          <div>
+            <LoadingSkeleton
+              isLoading
+              className="h-[1.3125rem] w-[50%] rounded"
```

Comment:
Hey @adam-strzelec, please change `w-[50%]` to `w-1/2`

Reply:
@CzarekDryl done